be/src/note/dto/update-note.input.ts

```diff
@@ -0,0 +1,9 @@
+import { InputType, Int, Field } from '@nestjs/graphql';
+import { Column } from 'typeorm';
+
+@InputType()
+export class UpdateNoteInput {
```

Comment:
The `UpdateNoteInput` and `CreateNoteInput` classes are the same currently. Might as well have only one for now